release_notes/release_notes.stex

```diff
@@ -121,6 +121,43 @@ Online versions of both books can be found at
 The type recovery pass has improved support for \scheme{abs} with a fixnum argument
 and added support for \scheme{1+}, \scheme{1-}, and \scheme{-1+}.
 
+\subsection{Foreign pointers to Scheme objects (10.2.0)}
+
+The \scheme{make-ftype-scheme-object-pointer} function facilitates a
+unified view of foreign-allocated addresses and addresses that are
+under control of the memory manager---especially addresses for
+bytevector and flvector content. Unlike extracting an address via
+\scheme{object->reference-address} and using it to construct a regular
+ftype pointer, the \scheme{make-ftype-scheme-object-pointer} function
+creates an ftype pointer that coopeartes with the memory manager to
```

Comment:
-> "cooperates"

Reply:
Thanks for all of these corrections! Now fixed.